release-notes/10.0/preview/preview1/dotnetmaui.md

```diff
@@ -1,15 +1,150 @@
 
 .NET MAUI updates in .NET 10:
 
 - [What's new in .NET MAUI in .NET 10](https://learn.microsoft.com/dotnet/maui/whats-new/dotnet-10) documentation.
 
+> **Known Issue:** building `net8.0-*` target frameworks with the .NET 10 SDK does not work in Preview 1.
 
-## Feature
+## CollectionView enhancements for iOS and Mac Catalyst
 
-This is something about the feature
+Two new handlers for `CollectionView` and `CarouselView` on iOS and Mac Catalyst that brought performance and stability improvements were available optionally in [.NET 9](https://learn.microsoft.com/dotnet/maui/whats-new/dotnet-9?view=net-maui-9.0#collectionview-and-carouselview). In this release they are now on by default. 
+
+You can _opt out_ of these handlers by adding the following code to your `MauiProgram` class if you want to revert back.
+
+```csharp
+#if IOS || MACCATALYST
+builder.ConfigureMauiHandlers(handlers =>
+{
+    handlers.AddHandler<Microsoft.Maui.Controls.CollectionView, Microsoft.Maui.Controls.Handlers.Items.CollectionViewHandler>();
+    handlers.AddHandler<Microsoft.Maui.Controls.CarouselView, Microsoft.Maui.Controls.Handlers.Items.CarouselViewHandler>();
+});
+#endif
+```
+
+We are excited for you to give these new handlers a try. As a reminder, if you have .NET 9 applications you can try them today with the following code:
+
+```csharp
+#if IOS || MACCATALYST
+builder.ConfigureMauiHandlers(handlers =>
+{
+    handlers.AddHandler<Microsoft.Maui.Controls.CollectionView, Microsoft.Maui.Controls.Handlers.Items2.CollectionViewHandler2>();
+    handlers.AddHandler<Microsoft.Maui.Controls.CarouselView, Microsoft.Maui.Controls.Handlers.Items2.CarouselViewHandler2>();
+});
+#endif
+```
+
+## .NET for Android
+
+This release was focused on quality improvements. A detailed list can be found on [dotnet/android GitHub releases](https://github.com/dotnet/android/releases/).
+
+### Android 16 (Baklava) Beta 1
+
+Google has released [Beta 1](https://android-developers.googleblog.com/2025/01/first-beta-android16.html) of the Android 16 (API-36) SDK.  Support has been added for using these preview APIs.
+
+To target the Android 16 preview API:
+- Use the Android SDK Manager to download the Android 16 (Baklava) Platform
+- Update your project's `TargetFramework` to `net10.0-android36`
+
+### Updated Recommended Minimum Supported Android API
+
+The .NET for Android project templates have been updated to specify `24` ("Nougat") as the default `$(SupportedOSPlatformVersion)` instead of `21` ("Lollipop").  This prevents users from hitting "desugaring" runtime crashes when using Java default interface methods.
+
+While API-21 is still supported in .NET 10, we recommend updating existing projects to API-24 in order to avoid unexpected runtime errors.
+
+> Note: See the main [GitHub pull request](https://github.com/dotnet/android/pull/9656) for more information.
+
+### Building with JDK-21 is now supported 
+
+.NET for Android projects can now be built with JDK-21.

```

Comment:
OK? I guess maybe this is enough of a statement like this, but... Why is that important? Is it still the Microsoft specific one? Where do I get it? Is this additional or does 17 stop working and do I need to update? Will 17 stop working in the future? When?